### DISCUSSION TITLE
[Req] Cache/queue option for converting to m4b

### DISCUSSION THREAD

**Discussion by WarHawk8080** [advplyr/audiobookshelf#1844] [Ideas]
under the series tab, add the option to add each individual book to a a queue to convert...and have it run sequentially rather than having to select one, then convert, then another then convert...and have the system queue the conversions rather than trying to run them all at the same time over stressing the CPU.  

Oh and have a "Select all" option under the series window, there is a deselect all...a select all button would be nice ;)

**Comment by advplyr** [comment]
This is a feature request here https://github.com/advplyr/audiobookshelf/issues/1267

I saw you also commented on #1569
